### DISCUSSION TITLE
YI 1080P RFUS Camera --> yi-hack-Allwinner-v2 (0.2.6) goes online - offline - online - offline......

### DISCUSSION THREAD

**Discussion by Darth2de** [roleoroleo/yi-hack-Allwinner-v2#615] [Q&A]
Hello,
i have a YI 1080P (RFUS) Camera with 12.0.3xxx firmware (latest) and in normal mode the camera goes online and i will see the live video in the app at my phone (Everything works normaly).

Then i put a sd-card in the camera with the newest "yi-hack-allwinner-v2 (0.2.6)" firmware into the slot.

When i boot the camera i can load via browser 192.168.xx.xx:8080 the new site. After 4 - 7 seconds, the camera goes offline. Then i wait 2-4 seconds the camera goes online. Then it goes online, after then it goes offline, online, offline......

![online-offline](https://user-images.githubusercontent.com/131043435/232597863-e5b85cc0-4930-4bfc-8ecf-cd76285a1dc6.jpg)

i tried a other SD-Card but it is the same.

I see the status led blue, then it goes off, then it blinks orange, the it goes blue, then it is off, then it blinks orange .........

Then i have switch everything "off" (Audio, telnet, SSH .... ) but still the same.

Hope, that someone can help me
(sorry, for my english)

regards
Darth2de

**Comment by roleoroleo** [comment]
The cam is in boot loop, it restarts during the boot process.
What's the exact firmware version? 12.....
What's the model suffix?

**Comment by Darth2de** [comment]
Hello,
thank you for your question.

 - Here the details:
 - Firmware Version | 0.2.6
 - Base Version  12.0.35.8_202211241029
 - Model Suffix | b091qp

I hope that this information is ok for you.

**Reply by roleoroleo** [reply]
Strange combination of model and firmware version.
Are you sure that it's a b091qp.
It looks like a y291ga, based on firmware version.

**Comment by Darth2de** [comment]
Hello,
this information was copied from yi-hack (first site) when i look via browser at 192.168.xx.xx:8080
![yi-camerainformation](https://user-images.githubusercontent.com/131043435/233129679-9d401820-5e65-49c8-82fe-d5fecf59e554.jpg)

**Comment by roleoroleo** [comment]
This could be the problem.
Try to use the hack for y291ga.

**Reply by Darth2de** [reply]
Thank you for answering. 

Where can i find rhe hack for y291ga?

**Reply by roleoroleo** [reply]
https://github.com/roleoroleo/yi-hack-Allwinner-v2/releases

**Reply by Darth2de** [reply]
Umpf. My fault. I have to look in „show more“ below. 
thank you and i will try this afternoon 👍

**Comment by Darth2de** [comment]
Ok, now it works and we can close this discussion. Many many many rhanks to you. Verry helpfull. 
with this Firmware it works fine
